cpu/memory.go

```diff
@@ -0,0 +1,87 @@
+package cpu
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io"
+)
+
+// Memory represents Gameboy Memory
+type Memory struct {
+	buffer [0x10000]byte
```

Comment:
Add a comment as to how you got that

Reply:
That's the max amount of memory cells a 16-bit bus can address.